model-optimizer/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -12,6 +16,8 @@
 from mo.utils.utils import refer_to_faq_msg
 from mo.utils.version import get_version
 
+defuse_stdlib()
```

Comment:
could you please explain what this routine is doing and why isn't it placed inside function?
Comment may be helpful

Reply:
Reworked for a better solution